app/javascript/packs/admin_users/user.jsx

```diff
@@ -282,6 +300,24 @@ class User extends React.Component {
     }
   }
 
+  handleCommentChange(event) {
+    this.setState({ body: event.target.value });
+  }
+
+  addNewComment() {
+    Fetch.postJSON('/api/v1/admin/comments', {
+        body: this.state.body,
+        receiver_id: this.props.user.id,
+    })
+      .then(() => {
+        $("input[type=text], textarea").val("");
```

Comment:
it is better to set the state values to the inputs and flush them.

Here are the docs how to use react with forms:
https://reactjs.org/docs/forms.html